components/Step4/Step4.tsx

```diff
@@ -0,0 +1,118 @@
+import React, {useEffect, useState} from 'react';
+import {Pressable, ScrollView, Text, View} from 'react-native';
+import {Input} from '../../components';
+import {selectWizard, updateUserData} from '../../redux/reducers/WizardData';
+import {useDispatch, useSelector} from 'react-redux';
+import {fitnessWorkouts} from '../../utils/fitnessConstants';
+import {FontAwesomeIcon} from '@fortawesome/react-native-fontawesome';
+import {faCheck} from '@fortawesome/free-solid-svg-icons';
+import {Colors} from '../../utils/colors';
+import {wizard} from '../../utils/constants';
+
+import globalStyle from '../../assets/styles/globalStyle';
+import style from './style';
+
+const Step4 = () => {
+  const dispatch = useDispatch();
+  const [customWorkoutValue, setCustomWorkoutValue] = useState('');
+  const wizardData = useSelector(selectWizard);
+  const [fitnessCategoriesWorkout, setFitnessCategoriesWorkout] = useState<
+    Array<{label: string; checked: boolean}>
```

Comment:
Sometimes you use `string[]`, sometimes you use `Array<string>` it's very inconsistent and it looks like it's written by another person. Try and keep it the same.